index.bs

```diff
@@ -29,11 +29,11 @@ If you wish to make comments regarding this document, please send them to <a hre
 
 <h2 id="intro">Introduction</h2>
 
-Given a complex responsive layout, developers often require granular control over how the contents of an individual module will respond relative to the size of their parent container rather than the viewport size. This limitation conflicts with the goal of creating modular, independent components, often requiring a number of redundant CSS, complex exception cases, and workarounds, and the problem compounds itself depending on how dramatically a module adapts at each of its breakpoints.
+Given a complex responsive layout, developers often require granular control over how the contents of an individual module will respond to the given module size rather than the viewport size. This limitation conflicts with the goal of creating modular, independent components, often requiring a number of redundant CSS, complex exception cases, and workarounds, and the problem compounds itself depending on how dramatically a module adapts at each of its breakpoints.
```

Comment:
I would phrase this as:
```
...an individual module will respond to changes to its own, or a parent module's size, rather than the viewport's size...
```

My problem with simply saying `an individual module will respond to the given module size` implies that such a solution would only be able to change an element's style based on it's own size.

That's the "element query" idea, afaik, where in contrast an element in the "container query" sense would be able to adjust it's own style based on a parent containing element's size.

@tomhodgins @marcoscaceres I think that's an important distinction.

Also not sure why we're talking about modules here instead of elements? 🤔

Reply:
You've flagged an important distinction.

I think `an individual element will respond to the given parent element dimensions` perhaps reads better?

Reply:
Yeah, but I also don't want to rule out elements changing their styles based on their own size.